### ISSUE TITLE
Status check email reports DS record added and removed

### ISSUE THREAD

**Issue by myfirstnameispaul** [mail-in-a-box/mailinabox#2003]
Daily receiving the following status checks change notice email, but each day for different domains:

```
example.com -- Added
=================
   Key Tag: 49514, Algorithm: 13, Digest Type: 2, Digest: e779aee5c93fb01ee708a7427e0ae8d1379a2b4cd62d81a1157e89a9ea6710a0

example.com -- Removed
===================
   Key Tag: 49514, Algorithm: 13, Digest Type: 2, Digest: e779aee5c93fb01ee708a7427e0ae8d1379a2b4cd62d81a1157e89a9ea6710a0

```
So far it is domains using external DNS and the records match the external DNS server DS records.

**Comment by Dibbyo456** [comment]
Hey, I am having the exact same issue. Any fix?

**Comment by myfirstnameispaul** [comment]
- [x] Mark read

**Comment by myfirstnameispaul** [comment]
This is fixed by #2109.
